scripts/prepare-release

```diff
@@ -61,7 +61,7 @@ if [[ "${new_version}" != *"-rc"* ]]; then
   perl -i -pe "s/${last_version}/${new_version}/g" "${source_dir}/README.md"
   pushd "${source_dir}/web"
   yarn install --frozen-lockfile
-  yarn docusaurus docs:version "${new_version_major_minor}"
+  yarn docusaurus docs:version "${new_version_major_minor}" || true
```

Comment:
How can this fail and still be fine?

Reply:
It's a leftover I yet have to remove from playing with the minor version override for the docs, which apparently doesn't work when there haven't been any changes to the docs between patch releases. For actually being able to run the script I had to put this in here temporarily.

Reply:
But yeah, review process working as intended here—thanks for taking a look at the diff. 👀

Reply:
I managed to fix this so it now always properly updates the docs for patch version bumps.